packages/react-server-cli/README.md

```diff
@@ -43,11 +43,39 @@ It's rare to see a project these days in the JavaScript world that isn't at leas
 To take advantage of the Babel compilation, you need to install the Babel plugins and presets you want and reference them in a `.babelrc` file in your code directory. For more on the `.babelrc` format, see [its documentation here](https://babeljs.io/docs/usage/babelrc/).
 
 ##Options
-Smart defaults are the goal, and `react-server-cli` has two base modes: **development** and **production**. `react-server-cli` will determine which base mode it's in by looking at (in order):
+Smart defaults are the goal, and `react-server-cli` has two base modes: **development** and **production**. `react-server-cli` will determine which base mode it's in by looking at the NODE_ENV environment variable. If it's not "production", then `react-server-cli` will assume we are in development mode.
 
-1. If the `--production` flag was sent in, it's **production**.
-1. If `process.env.NODE_ENV` is `'production'`, it's **production**.
-1. Otherwise, the base mode is **development**.
+###Ways to add options
+
+There are three ways to pass options to the CLI, through the command line, `.reactserverrc` JSON files, or as a `reactServer` entry in `package.json` files. Once `react-server` finds one of these sources of options, it will not search for more. It searches for options this way:
```

Comment:
I think this wording isn't quite right anymore with your latest changes.  Command-line options no longer cause `react-server` to stop searching for additional sources of options.  Same with the pseudo-code below.